docs/cicd.md

```diff
@@ -0,0 +1,194 @@
+# Continuous Integration and Delivery
+
+## Content
+
+- [Workflows](#workflows)
+- [Failed Delivery](#failed-delivery)
+- [Deliver a new release by triggering the CI](#triggering-a-delivery)
+- [Services](#services)
+- [Fastlane](#fastlane)
+- [Determining the next version](#determining-the-next-version)
+- [Environment variables](#environment-variables-and-dependencies)
+## Workflows
+
+Several workflows exist for different purposes:
+
+| Workflow                                   | Schedule/Trigger | Checks              | native delivery | backend_administration delivery | Version bump       |
+|--------------------------------------------|------------------|---------------------|-----------------|---------------------------------|--------------------|
+| commit                                     | commits of PRs   | :heavy_check_mark:  | :x:             | :x:                             | :x:                |
+| commit_main                                | commits on main  | :heavy_check_mark:  | :x:             | :x:                             | :x:                |
```

Comment:
:thought_balloon: unrelated to this PR, but any reason why we run checks on main as well? should never happen that we merge anything were checks do not succeed

Reply:
yes i think its not really needed. feel free to create an issue and remove it!